### PR TITLE
[Utopia-511] Update button hover states to match branding guidelines

### PR DESCRIPTION
**Pull request by antsand**
# Description

This PR includes the following proposed change(s):

- Update the button hover state
- Remove the progress bar component
- Jira Ticket # Utopia-511

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/sass/_buttons.scss [hunk 1]
```diff
@@ -93,26 +102,31 @@ a:hover {
     background-color: transparent;
 
     --text-color: white;
-
+
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:105]: empty spaces

File: src/frontend/src/sass/_buttons.scss [hunk 2]
```diff
@@ -93,26 +102,31 @@ a:hover {
...
+  }
 }
 
 /* Secnodary Dark styling */
 .bcgovbtn__secondary--dark {
   background-color: transparent;
 
   --text-color: #{$text-light};
-
+
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_buttons.scss:122]: same

**Comment by binggaofw** [comment]
Just a reminder you forgot update pr template
{List all the changes, if possible add the jira ticket #}

**Comment by binggaofw** [comment]
under hooks folder there is a file named useProgressBar, i think you can delete it as well

**Pull request merged**